### ISSUE TITLE
Wind correction angle

### ISSUE THREAD

**Issue by AntoineLeo** [albar965/littlenavmap#447]
Add the wind correction angle in the flight plan legs calculated from weather data along the flight route.

**Comment by albar965** [comment]
I suppose you're aware that this is not very accurate since the wind changes along a leg.
But I can at least add the angle at the start of the leg.

**Comment by AntoineLeo** [comment]
You are right. I think it's particularly useful when navigating only with NDB or VOR equipment.

**Comment by rvernica** [comment]
This would also be useful for VFR flight plans. Moreover, if the legs are 10-20nm then there is usually not a major wind change.

**Comment by mlipas** [comment]
Hey. Checking if this may be re added to a future release.

**Comment by albar965** [comment]
> Hey. Checking if this may be re added to a future release.

Yes but not sure which one. May take a while.

**Comment by mlipas** [comment]
Thanks. I am looking forward to having true/magnetic heading info, as this extremely important for those navigating via dead reckoning (although I understand such info may not be relevant for those navigating via GPS/VOR). 

Sample view of navlog of Skyvector with such info:
![image](https://user-images.githubusercontent.com/125236379/219817409-c95d434b-2475-40a7-b5af-7a808226e7a4.png)

**Comment by albar965** [comment]
Will consider this. I can show the WCA along with the start course at the beginning of the leg.
